### DISCUSSION TITLE
[Q&A] Unable to Turn off HTTPD

### DISCUSSION THREAD

**Discussion by clivinghouse** [glpi-project/glpi-agent#724] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

I am running GLPI Agent 1.10, I am trying to turn off the local web service by executing the following command. 
`.\glpi-agent.bat --no-httpd`

The agent is already installed.

I am using Windows 11 23H2
Thanks!

**Comment by g-bougard** [comment, marked as answer]
Hi @clivinghouse 

glpi-agent configuration is stored in registry under `HK_LM\SOFTWARE\Glpi-Agent`. You can change there the value for `no-httpd` and then restart the service.

**Reply by clivinghouse** [reply]
That did it. Thank you @g-bougard!
